lunar/distribution.yaml

```diff
@@ -238,11 +238,20 @@ repositories:
       version: kinetic-devel
     status: maintained
   catkin_pip:
+    doc:
+      type: git
+      url: https://github.com/pyros-dev/catkin_pip.git
+      version: devel
     release:
       tags:
         release: release/lunar/{package}/{version}
       url: https://github.com/pyros-dev/catkin_pip-release.git
-      version: 0.2.2-0
+      version: 0.2.3-0
+    source:
+      test_pull_requests: true
```

Comment:
The @ros-pull-request-builder user was invited to the repository (not the org unit) therefore the webhooks will need to be setup manually. That was currently not the case yet. Please let us know when that has happened and this can be merged.